lib/Model/Server.php

```diff
@@ -121,7 +121,7 @@ public static function swaggerFormats()
         'firewall' => 'firewall',
         'host' => 'host',
         'hostname' => 'hostname',
-        'ip_addresses' => 'ip_addresses',
+//        'ip_addresses' => 'ip_addresses',
```

Comment:
Why is this commented out 🤔

Reply:
(also commented out above and below)

Reply:
@Darep it's a temporary solution that will be removed within the next PR. Please, don't review this PR for now (since it's not finished yet).

Reply:
Gotcha, thanks! 👍 💯